js/views/search/Search.js

```diff
@@ -451,6 +456,17 @@ export default class extends baseVw {
     recordEvent('Discover_ChangeFilter');
   }
 
+  changeFilterAll(e) {
+    const targ = $(e.target);
+    this.formOverrides[targ.prop('name')] = targ.data('val').split(',') || '';
```

Comment:
Why default to an empty string here instead of an empty array? The latter keeps the types consistent.

Also, if there are no vals (edge case), then the data attribute will come in as `undefined` and this will bomb on the `split` call. So you should probably check that `targ.data('val')` is a string and only split if so.